statsmodels/nonparametric/tests/test_lowess.py

```diff
@@ -120,6 +120,16 @@ def test_options(self):
         assert_almost_equal(actual_lowess, actual_lowess1, decimal=13)
         assert_(actual_lowess.dtype is np.dtype(float))
 
+        # Test specifying xvals explicitly
```

Comment:
Could these be added as new tests of the feature, rather than to an existing test?